src/reference/asciidoc/whats-new.adoc

```diff
@@ -46,6 +46,13 @@ See <<request-reply>> for more information.
 (Also available in _1.4.4_ as setter on the `RabbitTemplate`).
 
 
+===== Listener Container Bean Names (XML)
+
+The `id` attribute on the `<listener-container/>` element is deprecated and ignored.
+Starting with this release, the `id` on the `<listener/>` child element us used alone to name the listener container bean.
+Normal Spring bean name overrides are applied; if a later `<listener/>` is parsed with the same `id` as an existing beam, the new definition will override the existing one.
```

Comment:
BTW, can we somehow to configure Travis to build `reference` as well?
This your is OK, but we really sometimes miss to check the docs locally...